cmux-tui/bindings/verify_release_environment_policy.py

```
@@ -0,0 +1,86 @@
+        return False
+    reviewer = entry.get("reviewer")
+    if not isinstance(reviewer, dict):
+        return False
+    for key in ("login", "slug", "name"):
+        value = reviewer.get(key)
+        if isinstance(value, str) and value.strip():
+            return True
+    return False
+
+
+def validate(document: object, expected_environment: str) -> str | None:
```

Ruff (0.16.1)

[warning] Too many return statements (10 > 6)

(PLR0911)